ui/lib/core/addon/components/form-field.hbs

```diff
@@ -306,8 +306,9 @@
           disabled={{and @attr.options.editDisabled (not @model.isNew)}}
           autocomplete="off"
           spellcheck="false"
-          value={{or (get @model this.valuePath) @attr.options.defaultValue}}
+          value={{get @model this.valuePath}}
```

Comment:
ember sets the `@model` attribute with whatever the `defaultValue` is - so this extra logic was redundant and causing overwrites [docs](https://guides.emberjs.com/release/models/defining-models/#toc_options)